src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -314,9 +321,11 @@ const WorkflowView = _.flow(
   ajaxCaller
 )(class WorkflowView extends Component {
   resetSelectionModel(value, selectedEntities = {}) {
+    const { entityMetadata } = this.state
     return {
       type: Utils.cond(
-        [_.endsWith('_set', value), () => EntitySelectionType.chooseSets],
+        [_.endsWith('_set', value) && Utils.log(_.includes(value, _.keys(entityMetadata))), () => EntitySelectionType.chooseSets], // TODO fix - this reads wrong on first load from Workflows page
```

Comment:
we need help with this - the value being printed to console is inconsistent depending on whether you arrive at this page by clicking on the workflow in the workflows tab, versus if you refresh or re-select the selection from the root entity type select.